src/NCronJob/Execution/StartupJobManager.cs

```diff
@@ -0,0 +1,33 @@
+namespace NCronJob;
+internal class StartupJobManager(JobRegistry jobRegistry)
```

Comment:
For reasons of consistency, I would refrain from using primary constructors.

Reply:
Ok, we should add an analyzer rule too.

Reply:
Well technically I don't think there is one. I marked `IDE0290` as none.

Not that I am absolutely against it, but I do think we need some shared guidelines and "enforce" them to some extent.

Reply:
So I am absolutely open to challenge the current coding standard/guidelines, especially if you feel that they are in your way and hinder productivity.

Reply:
I like the compactness of primary constructors. The only time they seem to get in the way is when using the source generated `LoggerMessage`.